src/services/read-from-disk.service.js

```diff
@@ -86,7 +86,7 @@ export function loadGuppyProjects(projectPathsInput: Array<string>) {
       projectPaths,
       function(projectPath, callback) {
         loadPackageJson(projectPath)
-          .then(json => callback(null, json))
+          .then(json => callback(null, { ...json, path: projectPath }))
```

Comment:
Why is this necessary?

Reply:
This is to be compatible with older users. When the old user opens, there may not be a 'byId' field, and I will store it when I read the item from disk.

Reply:
Gotcha. I replied earlier about not worrying about this. So I think you can remove this bit from here.

Thanks for thinking of this though!